composer.json

```diff
@@ -0,0 +1,36 @@
+{
+    "name": "integrated/subscription-bundle",
+    "type": "symfony-bundle",
+    "description": "Integrated Subscription Bundle",
+    "keywords": ["Integrated", "subscription"],
+    "homepage": "http://www.integratedfordevelopers.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "e-Active",
+            "homepage": "http://www.e-active.nl"
+        }
+    ],
+    "minimum-stability": "dev",
+    "prefer-stable": true,
+    "require": {
+        "php": ">=5.4",
+        "symfony/symfony": ">=2.4",
+        "symfony/framework-bundle": ">=2.2",
+        "sensio/framework-extra-bundle": "~2.3",
+        "doctrine/mongodb": ">=1.0.0",
```

Comment:
why do you require odm and mongo? I think most requirements stated here are actually not really required, plz check